qmxgraph/widget.py

```diff
@@ -183,9 +192,11 @@ def set_error_bridge(self, bridge):
 
         :param ErrorHandlingBridge bridge: Handler for errors.
         """
-        self._error_bridge = bridge
-        if self.is_loaded():
-            self._web_view.add_to_js_window('bridge_error_handler', bridge)
+        try:
+            self._error_bridge.on_error.disconnect(bridge.on_error)
+        except TypeError:
```

Comment:
Use `_silent_disconnect` instead (probably better to rename it to `silent_disconnect` also)